lib/types/geoman-minimal.d.ts

```
@@ -0,0 +1,120 @@
+  // Custom properties added to layers
+  interface Layer {
+    _shapeId?: string;
+    _textDirty?: boolean;
+    _url?: string;
+    toGeoJSON?(): any;
+    getLatLng?(): L.LatLng;
+    getLayers?(): Layer[];
+    feature?: {
+      properties?: {
+        _shapeId?: string;
+        [key: string]: any;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)